cuqi/experimental/mcmc/_laplace_approximation.py

```diff
@@ -0,0 +1,155 @@
+import scipy as sp
+import numpy as np
+import cuqi
+from cuqi.solver import CGLS
+from cuqi.experimental.mcmc import SamplerNew
+
+class UGLANew(SamplerNew):
```

Comment:
Perhaps this was dicussed in a daily update at some point, but I was hoping that LinearRTO and UGLA could use the same base class. If not in this PR, lets at least create an issue for it.

Reply:
I see your point: UGLA shares a lot of code with RTOs except for the additional approximation step. For now, I create  an issue for it #410.